doc/articles/Guides/CustomPlayer.md

```diff
@@ -0,0 +1,79 @@
+# Customizable Players
+
+In many games you want to customize character creation.  You may want to pick the color of the hair, eyes, skin, height, race, etc.
+
+By default Mirror will instantiate the player for you, which might prevent you from customizing it. However Mirror provides the option of overriding player creation and customize it.
+
+1) Create a class that extends NetworkManager if you have not done so. For example:
+```cs
+public class MMONetworkManager : NetworkManager
+{
+    ...
+}
+```
+and use it as your Network manager.
+2) Open your Network Manager in the inspector and disable the "Auto Create Player" boolean.
```

Comment:
```suggestion
2) Open your Network Manager in the inspector and disable the "Auto Create Player" boolean.  
```
You need 2 trailing spaces at the end of the line for the numbered list to format correctly.